plugins/pastefromword/filter/default.js

```diff
@@ -893,6 +893,13 @@
 				// Insert first known list item before the list wrapper.
 				innermostContainer.insertBefore( list[ 0 ] );
 
+				var keepZeroMargins = CKEDITOR.plugins.pastetools.getConfigValue( editor, 'keepZeroMargins' );
+
+				// Preserve keeping list margins zero when pasteTools_keepZeroMargins is ON. #5316
+				if ( keepZeroMargins ) {
+					innermostContainer.attributes.style = firstLevel1Element.attributes[ 'cke-list-style-margins' ];
```

Comment:
Is that temporary attribute really needed? From my quick test, it seems that `firstLevel1Element` styles contain `margin` styles at this point and can be used directly instead.